src/main/java/core/basesyntax/repoter/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.repoter;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
The class should not start with an empty line. Please remove the empty line above the class declaration.